repository/src/main/resources/db/changelog/db.changelog-0.0.13.xml

```diff
@@ -3,8 +3,9 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet id="13" author="ccd@hmcts.net">
+        <validCheckSum>8:cecbe4dfadf0450889899e86fde0f849</validCheckSum>
```

Comment:
how you know this is the valid check sum?

Reply:
Each file the check sum is different..the check sum is valid and is calculated by the liquidate..